app/scripts/views/settings.js

```diff
@@ -85,31 +85,6 @@ define(function (require, exports, module) {
       'navigate-from-child-view': '_onNavigateFromChildView'
     },
 
-    _initializeSubPanels (options) {
```

Comment:
I tried to simplify this a bit by extracting the logical units into their own functions.